src/LandPlanner.cpp

```diff
@@ -73,7 +81,8 @@ bool LandPlanner::prepareForTakeover(const ros::Time& time)
     }
 
     requested_height_ = transform.transform.translation.z;
-
+    // height determined by the ratio of landing accelerations
+    cushion_height_ = std::min(0.5 * (std::pow(descend_rate_,2)/cushion_acceleration_), requested_height_ * ( 1 - ( 1 / ( 1 - descend_acceleration_/cushion_acceleration_))));
```

Comment:
Can this be broken into multiple lines? Typically we try to limit line width to 80 characters.

Reply:
Is this equation for a worse case scenario of hitting the ground with zero velocity? Because with the cushion_rate limiter this will always be more conservative then it needs to be. However it won't be by a lot and I think its a great approximation.